tests/Makefile

```diff
@@ -62,3 +71,22 @@ unittest_prioq.o: \
 ../compile unittest_prioq.c
 	../compile unittest_prioq.c -I.. \
 	`pkg-config --cflags check`
+
+testable:
+	mkdir -p testable
+	cd ..; for i in *; do [ -f "$$i" ] && cp $$i tests/testable; done
```

Comment:
This doesn't seem right. If you create ```testable``` from here, then you want to copy the things to ```testable``` as well, not ```tests/testable```. Hint: if you want to copy to a directory, then always write ```testable/```, this would bail out if the target given is not a dir and would not copy all files over each other to the same name.